examples/postSubmissionApplication/data/planningPermission/major.ts

```diff
@@ -0,0 +1,185 @@
+        status: 'determined',
+      },
+      localPlanningAuthority: {
+        commentsAcceptedUntilDecision: true,
+      },
+      submission: {
+        submittedAt: '2025-02-05T11:33:56.009Z',
+      },
+      validation: {
+        receivedAt: '2025-02-05T11:33:56.009Z',
+        validatedAt: '2025-02-05T12:33:56.009Z',
+        isValid: true,
+      },
+      consultation: {
+        startDate: '2025-02-06',
+        endDate: '2025-03-08',
+        siteNotice: true,
+      },
+      assessment: {
+        councilDecision: 'refused',
+      },
+      appeal: {
+        lodgedDate: '2025-03-20',
+        reason:
+          "We don't believe the council took into consideration the environmental impact alleviation approach during their assessment.",
+        validatedDate: '2025-03-21',
+        startedDate: '2025-03-22',
+      },
+      caseOfficer: {
+        name: 'Casey Officer',
+      },
+    },
+    comments: {
+      public: [
+        {
+          sentiment: 'neutral',
+          comment: [
+            {
+              topic: 'design',
+              question:
+                'Comment on the design, size or height of new buildings or extensions',
+              comment: 'lalala',
+            },
+            {
+              topic: 'use',
+              question:
+                'Comment on the use and function of the proposed development',
+              comment: 'lalala',
+            },
+            {
+              topic: 'light',
+              question: 'Comment on impacts on natural light',
+              comment: 'lalala',
+            },
+            {
+              topic: 'privacy',
+              question: 'Comment on impacts to the privacy of neighbours',
+              comment: 'lalala',
+            },
+            {
+              topic: 'access',
+              question: "Comment on impacts on disabled persons' access",
+              comment: 'lalala',
+            },
+            {
+              topic: 'noise',
+              question: 'Comment on any noise from new uses',
+              comment: 'lalala',
+            },
+            {
+              topic: 'traffic',
+              question: 'Comment on impacts to traffic, parking or road safety',
+              comment: 'lalala',
+            },
+            {
+              topic: 'other',
+              question: 'Comment on other things',
+              comment: 'lalala',
+            },
+          ],
+          author: {
+            name: {
+              singleLine: 'John Doe',
+            },
+          },
+          metadata: {
+            submittedAt: '2025-01-27T12:50:49+0000',
+            publishedAt: '2025-01-27T12:50:49+0000',
+            validAt: '2025-01-27T12:50:49+0000',
+          },
+        },
+      ],
+      specialist: [
+        {
+          sentiment: 'Accepted',
+          constraint: {
+            name: 'Article 4 Direction Area',
+            url: 'https://www.planning.data.gov.uk/dataset/article-4-direction',
```

Comment:
nit: for constraints, is it possible to use Planning Data's "entity" URL rather than the general "dataset" here? 

For PlanX, this is the difference of being able to say "At least one Article 4 Direction applies" versus "This specific A4 direction applies" and we always favor/send the latter. (Especially because constraint name to source dataset can be a 1:many relationship!)

Reply:
aha! I knew I'd seen it somewhere in the code! I've switched it to use `Entity` type!

Reply:
ok I ran into a clash with this - `PlanningConstraint` is in shared and prototypeApplication so I've added a weird temporary one for now until we understand what the sentiment comments are doing with the constraints.